libs/api/src/lib/pagination.args.ts

```diff
@@ -0,0 +1,10 @@
+import {ArgsType, Field, Int} from '@nestjs/graphql'
+
+@ArgsType()
+export class PaginationArgs {
+  @Field(type => Int)
+  skip = 0
```

Comment:
Oh does this work? I usually use {defaultValue: 0} in the @Field, because I think it's now required to include in the GraphqlSchema